solidity/contracts/SisypheanExchange.sol

```diff
@@ -199,10 +199,12 @@ contract SisypheanExchange is ForkedERC1155 {
 		Universe memory universe = universes[_universeId];
 		require(universe.forkingMarket == 0, "Universe is forked");
 		MarketResolutionData memory marketResolutionData = marketResolutions[_universeId][_marketId];
+		uint8 initialReportOutcome = marketResolutionData.outcome;
```

Comment:
Is Solidity not smart enough to do this optimization on its own (put `marketResolutionData.outcome` on the stack instead of memory when it is merely read twice in the function)?

Reply:
I think it is. Can adjust this